### DISCUSSION TITLE
Método para generar una lista del abecedario

### DISCUSSION THREAD

**Discussion by naomipoclava2021** [santiemanuel/milProgramadoresQA#11] [Preguntas y Respuestas] [answered]
Necesito generar una lista con las letras del abecedario, me gustaria saber si algun metodo.

**Comment by santiemanuel** [comment, marked as answer]
Tenés varias formas de hacerlo. La más sencilla es importar `string`

## Con librería string: 
```py
import string

alfabeto = string.ascii_lowercase # o uppercase
listaletras = list(alfabeto)
```

## Con una cadena

```py
alfabeto = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
listaletras = list(alfabeto)
```

## Con listas por comprensión

### Para mayúsculas
```py
listaletras = [chr(i) for i in range(65, 91)]
```

### Para minúsculas
```py
listaletras = [chr(i) for i in range(97, 123)]
```

La función `chr(int)` devuelve un caracter ascii para la posición pasada como parámetro entero. Acá usamos el rango de valores ascii para las letras.
